packages/web-components/fast-components/src/switch/switch.styles.ts

```diff
@@ -57,214 +51,159 @@ export const switchStyles: (
-
-    .checked-indicator {
+    slot[name="switch"] {
         position: absolute;
-        top: 5px;
-        bottom: 5px;
-        background: ${neutralForegroundRest};
-        border-radius: calc(${controlCornerRadius} * 1px);
+        display: flex;
+        border: 1px solid transparent; /* Spacing included in the transform reference box */
+        fill: ${neutralForegroundRest};
         transition: all 0.2s ease-in-out;
     }
-
     .status-message {
         color: ${neutralForegroundRest};
         cursor: pointer;
         font-size: ${typeRampBaseFontSize};
         line-height: ${typeRampBaseLineHeight};
     }
-
-    :host([disabled]) .status-message,
-    :host([readonly]) .status-message {
-        cursor: ${disabledCursor};
-    }
-
-    .label {
-        color: ${neutralForegroundRest};
-
-        ${
-            /* Need to discuss with Brian how HorizontalSpacingNumber can work. https://github.com/microsoft/fast/issues/2766 */ ""
-        } margin-inline-end: calc(${designUnit} * 2px + 2px);
-        font-size: ${typeRampBaseFontSize};
-        line-height: ${typeRampBaseLineHeight};
-        cursor: pointer;
-    }
-
     .label__hidden {
         display: none;
         visibility: hidden;
     }
-
-    ::slotted(*) {
-        ${
-            /* Need to discuss with Brian how HorizontalSpacingNumber can work. https://github.com/microsoft/fast/issues/2766 */ ""
-        } margin-inline-start: calc(${designUnit} * 2px + 2px);
-    }
-
-    :host([aria-checked="true"]) .checked-indicator {
-        background: ${foregroundOnAccentRest};
-    }
-
-    :host([aria-checked="true"]) .switch {
-        background: ${accentFillRest};
-        border-color: ${accentFillRest};
-    }
-
-    :host([aria-checked="true"]:not([disabled])) .switch:hover {
-        background: ${accentFillHover};
-        border-color: ${accentFillHover};
-    }
-
-    :host([aria-checked="true"]:not([disabled])) .switch:hover .checked-indicator {
-        background: ${foregroundOnAccentHover};
-    }
-
-    :host([aria-checked="true"]:not([disabled])) .switch:active {
-        background: ${accentFillActive};
-        border-color: ${accentFillActive};
+    .label {
+        color: ${neutralForegroundRest};
+        font-size: ${typeRampMinus1FontSize};
+        line-height: ${typeRampMinus1LineHeight};
+        margin-inline-end: calc(${designUnit} * 2px + 2px);
+        cursor: pointer;
     }
-
-    :host([aria-checked="true"]:not([disabled])) .switch:active .checked-indicator {
-        background: ${foregroundOnAccentActive};
+    .status-message {
+        margin-inline-start: calc(${designUnit} * 2px + 2px);
     }
-
-    :host([aria-checked="true"]:${focusVisible}:not([disabled])) .switch {
-        box-shadow: 0 0 0 2px ${fillColor}, 0 0 0 4px ${focusStrokeOuter};
+    :host([aria-checked="true"]) .switch slot[name='switch'],
+    :host([aria-checked="true"]:enabled:hover) .switch slot[name='switch'] {
+        fill: ${neutralForegroundActive};
```

Comment:
Same question on targeting default slotted content vs. slotted content here - are we sure this catches a scenario when I slot my own SVG?